src/tools/data.js

```diff
@@ -169,6 +169,9 @@ const flattenPaths = (data, { preserve = {} } = {}) => {
 
 // A simpler, and memory-friendlier version of _.truncate()
 const simpleTruncate = (string, length, suffix) => {
+  if (string === undefined) {
```

Comment:
this was turning `undefined` into `''`, which the header formatter was yelling about.